### ISSUE TITLE
Non-Dismissable Warning Covering Content

### ISSUE THREAD

**Issue by andrewpeters9** [MetaMask/metamask-extension#18048]
**Reproduction:**
* Create a new account
* Skip the Recovery Phrase process in the account creation process
* Open the Popup Modal and you'll see the warning to backup your phrase
  * This phrase is non-dismissable and can cover up content

**Potential Solution:**
* Allow for dismissing
* To simplify the solution, the warning will reappear every time the modal is opened
  * I believe this would be the best solution as it doesn't require adding a preference etc. to the state
  * It also is a fairly important warning, hence deserves to be shown upon every popup
* Nit: While solving the ticket, center the warning (the padding on the left is almost 0, whereas the padding on the right is larger)

![image](https://user-images.githubusercontent.com/36251325/223709300-b81be09f-6bfa-48c7-b9aa-7cc0b3348f6f.png)

**Comment by Bspring305** [comment]
> **تولید مثل:**
> 
> * ایجاد یک حساب کاربری جدید
> * در فرآیند ایجاد حساب، از فرآیند عبارت بازیابی صرفنظر کنید
> * Popup Modal را باز کنید و اخطار پشتیبان گیری از عبارت خود را خواهید دید
>   
>   * این عبارت غیرقابل رد است و می تواند محتوا را پوشش دهد
> 
> **راه حل بالقوه:**
> 
> * اجازه اخراج
> * برای ساده کردن راه حل، هر بار که مدال باز می شود، هشدار دوباره ظاهر می شود
>   
>   * من معتقدم این بهترین راه حل خواهد بود زیرا نیازی به اضافه کردن اولویت و غیره به ایالت ندارد
>   * همچنین یک هشدار نسبتاً مهم است، از این رو شایسته است در هر پنجره نمایش داده شود
> * Nit: While solving the ticket, center the warning (the padding on the left is almost 0, whereas the padding on the right is larger)
> 
> ![تصویر](https://user-images.githubusercontent.com/36251325/223709300-b81be09f-6bfa-48c7-b9aa-7cc0b3348f6f.png)

**Comment by bschorchit** [comment]
This warning being non-dismissible is intended and there's already a preference in settings if you don't want to see that reminder anymore. cc: @AlexJupiter for visibility

![Screenshot 2023-03-08 at 14 15 04](https://user-images.githubusercontent.com/34968356/223782965-6ebcea25-30ea-4a9c-935a-28be99f6ecdd.png)

**Comment by danjm** [comment]
There is a way to dismiss, but we don't want to make it any easier for users to do that, because we want everyone to back up their seed phrase

This UX annoyance will also be improved for some users when we allow onboarding directly with hardware wallets.
